packages/design-system/src/components/web-components/ds-badge/ds-badge.stories.tsx

```diff
@@ -32,7 +32,6 @@ export default {
       description: {
         component: `For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/badge/).`,
       },
-      underlyingHtmlElements: ['span'],
```

Comment:
Removing these bits from our web components docs because it's inaccurate. I even tested this in our Astro project to confirm, but applying an attribute for `span` onto `ds-badge` doesn't result in that attribute being applied to the correct element.